src/main/java/com/snowflake/kafka/connector/internal/InternalUtils.java

```diff
@@ -316,7 +320,7 @@ public static Object backoffAndRetry(final SnowflakeTelemetryService telemetry,
       catch (Exception e)
       {
         finalException = e;
-        LOGGER.error(e.getMessage());
+        LOGGER.debug("Retry count:{} caught an exception for operation:{} with message:{}", iteration, operation, e.getMessage());
```

Comment:
I will change this in later PR.

Reply:
Done https://github.com/snowflakedb/snowflake-kafka-connector/pull/298/commits/513bb60e4085910e0204b4d61be18c900f656627